classes/Tutor.php

```diff
@@ -837,13 +856,134 @@ public static function create_database() {
 			PRIMARY KEY (withdraw_id)
 		) $charset_collate;";
 
+		$orders_table = "CREATE TABLE {$wpdb->prefix}tutor_orders (
+			id BIGINT(20) UNSIGNED AUTO_INCREMENT,
+			payment_id BIGINT(20) UNSIGNED COMMENT 'Unique payment id from payment gateway',
+			user_id BIGINT(20) UNSIGNED NOT NULL,
+			status ENUM('pending', 'completed', 'cancelled', 'refunded', 'partially-refunded') NOT NULL,
```

Comment:
This is hard to alter `ENUM` if you need to change the table structure. Better use varchar and use comment.

Reply:
We need to separate the payment status and the order status. One order could be `refunded` and could be `cancelled` at the same time.

Reply:
We may need to store the statuses like `pending, paid, failed, refunded, partially-refunded`